MediaBrowser.Controller/Sorting/SortExtensions.cs

```diff
@@ -30,5 +30,10 @@ public static IOrderedEnumerable<T> ThenByStringDescending<T>(this IOrderedEnume
         {
             return list.ThenByDescending(getName, _comparer);
         }
+
+        public static IEnumerable<string> Trimmed(this IEnumerable<string> values)
```

Comment:
This isn't the right place for this, maybe somewhere in Jellyfin.Extensions?

Reply:
Can do!

Reply:
Done